elements/pfe-navigation/documentation/implementation.md

```diff
@@ -0,0 +1,227 @@
+        />
+      </a>
+    </div>
+    <ul class="pfe-navigation__menu" id="pfe-navigation__menu">
+      <li class="pfe-navigation__menu-item">
+        <a href="#LINK-TO-CONTENT" class="pfe-navigation__menu-link">
+          Menu Link 1
+        </a>
+      </li>
+
+      <li class="pfe-navigation__menu-item">
+        <a href="#LINK-TO-CONTENT" class="pfe-navigation__menu-link">
+          Menu Link 2
+        </a>
+      </li>
+
+      <li class="pfe-navigation__menu-item">
+        <a href="#LINK-TO-CONTENT" class="pfe-navigation__menu-link">
+          Menu Link 3
+        </a>
+      </li>
+
+      <li class="pfe-navigation__menu-item">
+        <a href="#LINK-TO-CONTENT" class="pfe-navigation__menu-link">
+          Menu Link 4
+        </a>
+      </li>
+    </ul>
+
+  </nav>
+</pfe-navigation>
+```
+
+> Unfortunately we need to make sure all of the id's and classes are correct on this level for fall back styling, functionality may break if classes or id's are missing or incorrect.
+
+### Adding dropdowns to the menu
+To add a dropdown, add the following markup inside of an `<li class="pfe-navigation__menu-item">`, but after the `<a>`, like this:
+
+```html
+<li class="pfe-navigation__menu-item">
+  <a href="#LINK-TO-CONTENT" class="pfe-navigation__menu-link">
+    Menu Link 1
+  </a>
+
+  <!-- Dropdown markup -->
+  <div class="pfe-navigation__dropdown">
+  </div>
+</li>
+```
+
+#### Adding menu links to a dropdown
+
+If a group of links has a title, it's HTML should be as follows:
+
+```html
+<section>
+  <!-- Must be a heading tag, but can be any heading level that is appropriate for the page -->
+  <h2>Links title</h2>
+  <ul>
+    <li><a href="#LINK">Link text</a></li>
+    <li><a href="#LINK">Link text</a></li>
+    <li><a href="#LINK">Link text</a></li>
+    <li><a href="#LINK">Link text</a></li>
+  </ul>
+</section>
+```
+
+If a group of links **does not** have a title, it's markup should be as follows:
+
+```html
+<ul>
+  <li><a href="#LINK">Link text</a></li>
+  <li><a href="#LINK">Link text</a></li>
+  <li><a href="#LINK">Link text</a></li>
+  <li><a href="#LINK">Link text</a></li>
+</ul>
+```
+
+#### Making multi-column dropdown
+Dropdowns are full width and multi-column by default. Styling is handled by the web component, but there are layout classes that can be added to control the layout of the dropdown.
+
+The default layout of multi-column dropdowns is not intended to be used much, it's the default because it is a 'safe' default that won't result issues.
+
+To create a custom column layout in a dropdown, see [Custom Dropdown Layout Documentation](custom-dropdown-layout.md).
+
+##### Adding a footer to the multi-column dropdown
+To add a full width footer, add the following markup right before the dropdown's closing tag:
+```html
+<div class="pfe-navigation__footer">
+</div>
+```
+
+`<pfe-cta>` is available, which will provide red hat theming and behavior, so a footer's markup could be something like the following:
```

Comment:
```suggestion
`<pfe-cta>` is available, which will provide styles and behavior, so a footer's markup could be something like the following:
```

pfe-cta is not "Red Hat" theming per-say; it's part of the Design System.  I think we avoid saying it's Red Hat and focus on the PatternFly brand as much as possible to stay "upstream".